src/gatsby/gatsby-config/index.ts

```diff
@@ -127,12 +127,12 @@ export default {
           createReleaseNotesFeed({
             description: 'Release notes for MaxMind\'s GeoIP2 product line',
             title: 'GeoIP2 Release Notes',
-            url: '/geoip/release-notes',
+            url: '/geoip/release-notes/2022',
```

Comment:
@kevcenteno I hardcoded this similar to the reason in https://github.com/maxmind/dev-site/pull/317#discussion_r816188612. Was this the right thing to do?

Reply:
Actually, I'm not sure this is the best method because now we would have to update `/geoip/release-notes/rss.xml` to `/geoip/release-notes/2022/rss.xml` as well as the necessary redirects each year. I'll keep poking.